src/Containers-OrderPreservingDictionary/CTStandardOrderedDictionary.class.st

```diff
@@ -7,6 +7,23 @@ For example, the printing is using the insertion order
 
 	CTStandardOrderedDictionary orderedDictionaryABC printString 
 	>>> 'a CTStandardOrderedDictionary(#A->''A'' #B->''B'' #C->''C'')'
+
+I work mainly as a Dictionary except that I also store keys in an Array to keep track of the order of elements.
```

Comment:
What's the difference between this one and `CTOrderPreservingDictionary`?

Reply:
Only that it is raising an error